recipes/sourmash/meta.yaml

```diff
@@ -1,14 +1,14 @@
 {% set name = "sourmash" %}
-{% set version = "2.0.0a4" %}
-{% set hash = "9ec94badf31c1fd9f7564008aafebbc3aea174add98efa8e6b2b7b42b662ffe1" %}
+{% set version = "2.0.0a5" %}
+{% set hash = "611a6adcaa2c659a79bd0f111ad75fc25cb0ee0298e53addd1e7aeae00c62b0a" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   fn: {{ name|lower }}-{{ version }}.tar.gz
-  url: https://github.com/dib-lab/sourmash/archive/102b9eac5bf5bba41119fcf8b39bbebf4a09279a.tar.gz
+  url: https://github.com/dib-lab/sourmash/archive/790dddd65ca9ee599f8a83fd251a198ddc8e9352.tar.gz
```

Comment:
For the future, consider tagging the release on GitHub and refer to the tag here.